toolchains/freemint/packages/toolchain/build.sh

```diff
@@ -0,0 +1,20 @@
+#! /bin/sh
+
+# TODO: Use fixed revisions for binutils, gcc, mintlib, mintbin and fdlibm
+TOOLCHAIN_VERSION=b115a55c606cbf4c07a5fd199b2c2eb5b28c4725
+
+PACKAGE_DIR=$(CDPATH= cd -- "$(dirname -- "$0")" && pwd)
+HELPERS_DIR=$PACKAGE_DIR/../..
+. $HELPERS_DIR/functions.sh
+
+do_make_bdir
+
+do_http_fetch m68k-atari-mint-build "https://github.com/mikrosk/m68k-atari-mint-build/archive/${TOOLCHAIN_VERSION}.tar.gz" 'tar xzf'
+
+# TODO: Build the m68020-60 and 5475 toolchains as well
```

Comment:
Actually, I have now realised that you don't need to build those at all.

They are useful only for building native (68020+ / ColdFire) compilers. The usual way on our platform is to override CFLAGS / CXXFLAGS, i.e. adding `-m68020-60` / `-mcpu=5475`, usually along with `-fomit-frame-pointer` (which is not very useful and has a performance hit).

Another approach would be to override `ENV INSTALL_DIR` and `PREFIX` so they point to the proper `sys-root` for given CPU (i.e. no multilib setup but separated environment for every CPU) but `mintlib`, `gemlib` and others require some tweaks for such scenario (i.e. not to build their multilib variants what is done by default).